packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.34.3 (2024-07-19)
+
+- Fix Hardhat compile error when multiple return parameters are documented. ([#1048](https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/1048))
```

Comment:
Is it just Hardhat? I get #1047 got it through @tenderly/hardhat-tenderly because it depends on `@openzeppelin/upgrades-core`. I think compilation may be failing in Foundry too

Reply:
This code path is currently only active when compiling in Hardhat.

Reply:
I think this happens because the above project enables `@tenderly/hardhat-tenderly` in hardhat.config.ts, and `@tenderly/hardhat-tenderly` imports `@openzeppelin/hardhat-upgrades` [here](https://github.com/Tenderly/hardhat-tenderly/blob/d97dc377fa9659942e8fe6b9e9add64d9fd4eb1f/packages/tenderly-hardhat/src/tenderly/extender.ts#L1).

So this means `@tenderly/hardhat-tenderly` effectively inherits Hardhat Upgrade's compilation subtask override, which adds the namespaced compilation step whenever Hardhat compile is run.